Dockerfile

```diff
@@ -1,18 +1,19 @@
-FROM node:14.16.0
+ARG base_image=node:16-slim
```

Comment:
Was this agreed on with the dev team?

Reply:
No it's a suggestion because the existing version is too old. I tested the build with the new image. Will wait for their approval before merging.

Reply:
I think we just default to using 14.16. Maybe Jakob had a good reason, though.

Reply:
Feel free to revert the change if that's the case.